src/config/default.toml

```diff
@@ -1,3 +0,0 @@
-[meta]
-disable_recovery = true
```

Comment:
I guess we really want to disable recovery with RiseDev. 🤔

Reply:
I don't think so 🤔 Actually for my cases I am used to `risedev d full` and it keeps my tables when rebooting.